actions/websocket_actions.jsx

```diff
@@ -58,23 +58,30 @@ export function initialize() {
     if (config.WebsocketURL) {
         connUrl = config.WebsocketURL;
     } else {
-        connUrl = getSiteURL();
+        connUrl = new URL(getSiteURL());
```

Comment:
IE11 does not [support](https://caniuse.com/#feat=url) URL. Would this be a problem for us in that case?

Reply:
Looks like the polyfill that we use adds that in since the websocket connects fine for me using the text server with IE11